roles/server/tasks/go-server.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: apt install go-server
   sudo: yes
-  apt: pkg=go-server={{ GOCD_GO_VERSION }} state=present force=yes
+  apt: pkg=go-server{{ '=' + GOCD_GO_VERSION if GOCD_GO_VERSION != 'latest' else ''}} state=present force=yes
```

Comment:
Hmm I think that I was using this package in a different way than intended; namely I ran the Common role separately from the Server / Agent role (because they may be different machines). It looks like then the common role is run in the same play with server or agent, then GOCD_GO_VERSION=latest is replaced by the actual latest version, but this change does not pass on to the server/agent role when it is in a separate play.

I will also change the go-agent there none the less.

I cannot test yum as my disk is too full to launch another VM, so i will just change apt for now.